cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -59,10 +62,11 @@ class CallTest extends DocumentingTest {
       p(
```

Comment:
The definition of "standalone procedure call" seems to be getting confusing. It's not a single CALL clause anymore because you can have YIELD * (as per below). Does the query parameter feature still work with YIELD *? If not then this needs to be defined separately, otherwise a "standalone procedure call" can be a single CALL clause optionally followed by YIELD *.

Reply:
you mean the implicit argument thing, `"CALL org.neo4j.stream123 YIELD *"` vs `"CALL org.neo4j.stream123() YIELD *"`? Both those queries work fine

To make matters even more complicated, `USE db CALL bla()` is still a standalone call as well :P

Reply:
I meant this form:
```
:param databaseName=>'movies'
call dbms.database.state
```

Reply:
it does yes